static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -1180,6 +1182,24 @@ export class TraceTreeNode<T extends TraceTree.NodeValue> {
   }
 }
 
+export class MissingInstrumentationNode extends TraceTreeNode<TraceTree.MissingInstrumentationSpan> {
```

Comment:
@Abdkhan14 we store the previous and next span so that we can display some helpful info in the details panel